internal/util/which.go

```diff
@@ -0,0 +1,28 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package util
+
+import "os/exec"
+
+// check if binary is present in PATH
+func Which(binary string) error {
```

Comment:
(nit): Comments should begin with the name of the thing being described and end in a period, like:
```go
// Which checks if binary is present in PATH
func Which(binary string) error {
```

Refer to https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
ok, i will check comments and format them.